src/main/application/BorosTcf.js

```diff
@@ -3,36 +3,44 @@ import {inject} from '../core/ioc/ioc'
 import {GetVendorListUseCase} from './services/vendorlist/GetVendorListUseCase'
 import {LoadUserConsentUseCase} from './services/vendorconsent/LoadUserConsentUseCase'
 import {SaveUserConsentUseCase} from './services/vendorconsent/SaveUserConsentUseCase'
+import ChangeUiVisibleUseCase from './services/ui/ChangeUiVisibleUseCase'
```

Comment:
use named imports instead of default imports inside the lib to:
- keep coherence with the other classes
- avoid losing names when any refactor is done (default imports are not needed to be changed in refactors and it causes losing name sync)

Reply:
done.